Fix out of tree build errors and parallel make issues. The mkdir commands 
can race each other. The srcdir variable is a complete path so a ../ prefix makes
no sense.

Upstream-Status: Pending

RP 2013/03/21

Index: libuser-0.58/docs/Makefile.am
===================================================================
--- libuser-0.58.orig/docs/Makefile.am	2012-11-07 18:25:58.000000000 -0800
+++ libuser-0.58/docs/Makefile.am	2013-03-22 23:32:06.843518069 -0700
@@ -8,14 +8,14 @@
 
 all: sgml/libuser.txt sgml/libuser.html
 
-libuser.conf.5: $(srcdir)/libuser.conf.5.in Makefile
+libuser.conf.5: $(abs_srcdir)/libuser.conf.5.in Makefile
 	sed 's,@sysconfdir\@,$(sysconfdir),g' \
-		< $(srcdir)/libuser.conf.5.in > $@
+		< $(abs_srcdir)/libuser.conf.5.in > $@
 
-sgml/libuser.txt: $(srcdir)/sgml/libuser.sgml
-	[ -d sgml ] || mkdir sgml
-	cd sgml; sgml2txt ../$(srcdir)/sgml/libuser.sgml
+sgml/libuser.txt: $(abs_srcdir)/sgml/libuser.sgml
+	[ -d sgml ] || mkdir sgml || true
+	cd sgml; sgml2txt $(abs_srcdir)/sgml/libuser.sgml
 
-sgml/libuser.html: $(srcdir)/sgml/libuser.sgml
-	[ -d sgml ] || mkdir sgml
-	cd sgml; sgml2html ../$(srcdir)/sgml/libuser.sgml
+sgml/libuser.html: $(abs_srcdir)/sgml/libuser.sgml
+	[ -d sgml ] || mkdir sgml || true
+	cd sgml; sgml2html $(abs_srcdir)/sgml/libuser.sgml
